capture/configs/aes_sca_cw305.yaml

```diff
@@ -0,0 +1,54 @@
+target:
+  target_type: cw305
+  fpga_bitstream: "../objs/lowrisc_systems_chip_englishbreakfast_cw305_0.1.bit"
+  force_program_bitstream: True
+  fw_bin: "../objs/aes_serial_fpga_cw305.bin"
+  # target_clk_mult is a hardcoded value in the bitstream. Do not change.
+  target_clk_mult: 0.1
+  target_freq: 10000000
+  baudrate: 115200
+  output_len_bytes: 16
+  #protocol: "ujson"
```

Comment:
```suggestion
```
Do we maybe want to remove the ujson entry for now? As we currently only support simpleserial for CW305 targets, we might not want to give other users the impression that this is working?

Reply:
Good idea!